transformations/contextual_meaning_perturbation/requirements.txt

```diff
@@ -0,0 +1,4 @@
+transformers==4.8.2
+
+# for contextual_perturbation (DE) download DE SpaCy model:
+de-core-news-sm==2.2.5
```

Comment:
The build fails with the spacy model.

For adding spacy model as a dependency in requirements.txt, you may take a look here:
https://stackoverflow.com/questions/61702357/how-to-add-a-spacy-model-to-a-requirements-txt-file

Reply:
Ah I see, that makes sense. I removed it from the requirements.txt since it is not necessary to download the German model to use the transformation in general.